### DISCUSSION TITLE
Is it possible to hide a field from displaying in the preview pane?

### DISCUSSION THREAD

**Discussion by philiprenich** [decaporg/decap-cms#7279] [Q&A] [answered]
I'd like to hide certain fields, like the date and tags field, from the preview pane. They are metadata and I don't need to see them inline with the content I'm writing. It would also give me the freedom to place the fields in whatever order I'd like in the editor side, without them showing up oddly in the preview. Did I miss a configuration option or does this require building my own Preview component?

**Comment by martinjagodic** [comment, marked as answer]
@philiprenich there is no such config, you will have to build a custom preview component.

Sorry for the late reply, I hope it's still relevant.
